skpy/main.py

```diff
@@ -190,7 +190,7 @@ def cycle(self):
         """
         try:
             events = self.getEvents()
-        except requests.ConnectionError:
+        except (requests.ConnectionError,AttributeError) as e:
```

Comment:
Not a good idea to catch that, if you're seeing one then it's probably a bug in the event parsing.  Can you reproduce the error and drop a stack trace in the comments?